tests/FSharp.Compiler.ComponentTests/Conformance/LexicalFiltering/OffsideExceptions/OffsideExceptions.fs

```diff
@@ -474,6 +474,12 @@ raise (new Exception("exit 1"))
                         EndColumn = 40 }
               Message =
                "Unexpected end of quotation in expression. Expected incomplete structured construct at or before this point or other token." }
+            { Error = Error 3567
```

Comment:
Line 5 points to the "raise new Exception" code, I do not immediately see what is wrong with that?

Reply:
The parser can't parse the type inside the quotation (and it's expected), and with the new recovery rules it just goes a bit further and gives an additional error. Both existing errors and this added one can't help much anyway on such a broken sample.